.circleci/config.yml

```diff
@@ -46,8 +46,8 @@ jobs:
             . venv/bin/activate
             coverage run --source="." manage.py test
             coverage report --fail-under=83 -m
-            # pycodestyle ./openedx_external_enrollments
-            # pylint ./openedx_external_enrollments --rcfile=./setup.cfg
+            pycodestyle ./openedx_external_enrollments
```

Comment:
Can we include these commands in the Makefile? @andrey-canon

Reply:
not, why did you suggest that in the pr where the tests are included?

Reply:
Good question, I didn't notice. @andrey-canon

Reply:
So let's change it in a subsequent PR, do you agree? @andrey-canon

Reply:
ok